packages/interwovenkit-react/src/pages/wallet/tabs/portfolio/VaultSection.module.css

```
@@ -0,0 +1,226 @@
+  overflow: hidden;
+
+  &[data-state="open"] {
+    animation: slideDown var(--transition) ease-out;
+  }
+
+  &[data-state="closed"] {
+    animation: slideUp var(--transition) ease-out;
+  }
+}
+
+@keyframes slideDown {
```

Stylelint (17.13.0)

[error] Expected keyframe name "slideDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)